### ISSUE TITLE
Fluent UI Button component doesn't renders with React Testing Library render method.

### ISSUE THREAD

**Issue by dk511** [microsoft/fluentui#14049]
render method which belongs to React testing library throws below error

![image](https://user-images.githubusercontent.com/58889061/87608817-b964e980-c6b5-11ea-876a-1d6641b90975.png)

![image](https://user-images.githubusercontent.com/58889061/87609035-59227780-c6b6-11ea-916d-377438578608.png)

Note : Am using   "@fluentui/react-northstar": "^0.50.0",

Am also using Provider component with theme at the root. I am testing a simple button using react testing library. Why is it failing for "telemetry"?

**Comment by layershifter** [comment]
@dk511 I was able to repro this problem, it was also reported in #13836 and fixed by #14010. 0.50.0 (contains an issue) https://codesandbox.io/s/fluent-ui-issue-14049-b049r, in 0.51.4 it should run properly. Can you please check?

---

However, even with there is an issue related to JSDOM in `fontSizeUtility.ts` that I met in CodeSandbox with 0.51.4 🤔 

https://github.com/microsoft/fluentui/blob/46055c7b84750bda3d609ab8bf58bb53e4e952cd/packages/fluentui/react-northstar/src/utils/fontSizeUtility.ts#L10

There is a collision between native DOM and JSDOM as it provides own implementation of HTML elements, `Document` & `Window` objects:

```tsx
getComputedStyle === dom.window.getComputedStyle // false
dom.window.document.documentElement instanceof Element // 💣 false
```

As we are using `getComputedStyle()` directly it fails with:

```tsx
TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
```

It should not be an issue for Node environment, but can be annoying for CodeSandbox, a repro can be found on CSB:
https://codesandbox.io/s/fluent-ui-issue-14049-forked-sl7zo?file=/src/index.js

**Comment by layershifter** [comment]
> However, even with there is an issue related to JSDOM in `fontSizeUtility.ts` that I met in CodeSandbox with 0.51.4 🤔

Was fixed by #16368.
